src/components/Walkthrough/Walkthrough.js

```diff
@@ -0,0 +1,292 @@
+// @flow
+import * as React from 'react';
+import { Dimensions, StyleSheet, SafeAreaView, Platform } from 'react-native';
+import styled from 'styled-components/native';
+import { connect } from 'react-redux';
+import ExtraDimensions from 'react-native-extra-dimensions-android';
+import { NavigationActions } from 'react-navigation';
+import { getNavigationPathAndParamsState, navigate } from 'services/navigation';
+import Button from 'components/Button';
+import { Paragraph, BoldText } from 'components/Typography';
+import Toast from 'components/Toast';
+import { spacing, fontStyles } from 'utils/variables';
+import { endWalkthroughAction, setWaitingForStepIdAction, setActiveStepIdAction } from 'actions/walkthroughsActions';
+import type { Measurements, Steps } from 'reducers/walkthroughsReducer';
+import { WALKTHROUGH_TYPES } from 'constants/walkthroughConstants';
+import { themedColors } from 'utils/themes';
+import { hexToRgba } from 'utils/ui';
+import { WalkthroughTooltip } from './WalkthroughTooltip';
+
+const { width, height: h } = Dimensions.get('window');
```

Comment:
```suggestion
const { width, height: windowHeight } = Dimensions.get('window');
```
?

Reply:
Since I've only using h on the next line to set window height by platform (as I needed it adjusted for Android), I simply named it h. But yeah, I could rename it :)